projects/angular/src/emphasis/alert/_alert.clarity.scss

```
@@ -607,6 +662,52 @@
         line-height: $smallAlertCloseIconDims + variables.$clr_baselineRem_1px;
       }
     }
+
+    &.alert-lightweight {
+      background: none;
+      border: none;
+      .alert-items {
+        @include mixins.css-var(
+          gap,
+          clr-alert-lightweight-sm-padding,
+          variables.$clr_baselineRem_0_25 variables.$clr_baselineRem_5px,
+          variables.$clr-use-custom-properties
+        );
+        @include mixins.css-var(
+          padding,
+          clr-alert-lightweight-sm-padding,
+          variables.$clr_baselineRem_0_25 variables.$clr_baselineRem_5px,
+          variables.$clr-use-custom-properties
+        );
+      }
+      .alert-item {
+        min-height: 0;
+        .alert-text {
+          @include mixins.css-var(
+            font-size,
+            clr-alert-lightweight-sm-font-size,
+            alert-variables.$clr-alert-font-size,
+            variables.$clr-use-custom-properties
+          );
+          @include mixins.css-var(
+            line-height,
+            clr-alert-lightweight-sm-line-height,
+            variables.$clr_baselineRem_0_667,
+            variables.$clr-use-custom-properties
+          );
+        }
+      }
+      .close {
+        width: auto;
+        margin: 0;
+        @include mixins.css-var(
+          padding-right,
+          clr-alert-lightweight-sm-padding,
+          variables.$clr_baselineRem_0_25,
+          variables.$clr-use-custom-properties
+        );
+      }
+    }
```

Same as normal lightweight alerts. Close button is not supported.
```suggestion
      .close {
        display: none;
      }
    }
```

---

The close style would be inherited from the base rule added.